src/test/SearchDateInput.test.tsx

```diff
@@ -0,0 +1,52 @@
+import { render, screen } from '@testing-library/react'
+import SearchDateInput from '../components/search/SearchDateInput'
+
+test('Show correct label when rendered', () => {
+  const { container } = render(
```

Comment:
@Lumiere-Lodi should this not be wrapped in act? Getting warning errors

Reply:
it was happening in the search file. FIXED now. You can test again.